### PR TITLE
Add trivial echo-true job to the sandbox project

### PR DESCRIPTION
**Pull request by jesusaurus**
This trivial job simply echos 'true'. This will exercise nodepool instance
creation and deletion, as well as loading zuul job configuration.

### CODE REVIEW

File: roles/zuul/tasks/main.yml [hunk 1]
```diff
@@ -96,6 +96,11 @@
     owner: zuul
   notify: Restart zuul
 
+- name: Install job definitions
+  synchronize:
```
Comment (jamielennox) [inline roles/zuul/tasks/main.yml]: so synchronize doesn't work wih CI. Probably for now it's worth just doing a with_fileglob and copy.

Comment (nibalizer) [inline roles/zuul/tasks/main.yml]: yah

Comment (nibalizer) [inline roles/zuul/tasks/main.yml]: Looks like this got fixed, so we just need to rebase

File: roles/zuul/files/jobs/echotrue.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+job:
```
Comment (j2sol) [inline roles/zuul/files/jobs/echotrue.yaml]: This is crashing zuul-launcher with:

```
Traceback (most recent call last):
  File "/opt/venvs/zuul/bin/zuul-launcher", line 10, in <module>
    sys.exit(main())
  File "/opt/venvs/zuul/local/lib/python2.7/site-packages/zuul/cmd/launcher.py", line 118, in main
    server.main(False)
  File "/opt/venvs/zuul/local/lib/python2.7/site-packages/zuul/cmd/launcher.py", line 85, in main
    self.launcher.start()
  File "/opt/venvs/zuul/local/lib/python2.7/site-packages/zuul/launcher/ansiblelaunchserver.py", line 271, in start
    self.loadJobs()
  File "/opt/venvs/zuul/local/lib/python2.7/site-packages/zuul/launcher/ansiblelaunchserver.py", line 300, in loadJobs
    builder.load_files([path])
  File "/opt/venvs/zuul/local/lib/python2.7/site-packages/jenkins_jobs/builder.py", line 293, in load_files
    self.parser.parse(in_file)
  File "/opt/venvs/zuul/local/lib/python2.7/site-packages/jenkins_jobs/parser.py", line 128, in parse
    self.parse_fp(fp)
  File "/opt/venvs/zuul/local/lib/python2.7/site-packages/jenkins_jobs/parser.py", line 103, in parse_fp
    cls=type(data)))
jenkins_jobs.errors.JenkinsJobsException: The topmost collection in file '/var/lib/zuul/jobs/echotrue.yaml' must be a list, not a <class 'collections.OrderedDict'>
```

**Pull request merged**